setup.py

```diff
@@ -190,6 +190,12 @@ def build_extension(self, ext: Extension):
                 "depends": srcdeps_crc_common,
                 "gcc_x86_flags": ["-mssse3", "-msse4.1", "-mpclmul"],
             },
+            {
+                "sources": ["src/yencode/crc_folding_256.cc"],
+                "depends": srcdeps_crc_common,
+                "gcc_x86_flags": ["-mavx2", "-mvpclmulqdq", "-mpclmul"] if IS_X86 and autoconf_check(self.compiler, flag_check="-mvpclmulqdq") else [],
```

Comment:
On Windows this is always available if `/arch:AVX2` is supplied?
Because now the  `autoconf_check` is also executed on Windows, but I never made it work there so it's giving a bit of an ugly error: https://github.com/animetosho/sabyenc/runs/6502573699?check_suite_focus=true#step:5:82

Reply:
Ah missed that - good catch!  
Should be fixed in latest commit.

Reply:
👍 Thanks!